openlabcmd/openlabcmd/plugins/nodepool/router.py

```diff
@@ -0,0 +1,67 @@
+import subprocess
```

Comment:
I think router belongs to network. No need to create a new py file, just merge it inot network.py